x/concentrated-liquidity/README.md

```diff
@@ -147,128 +209,162 @@ $$price = (10^{4}) + (650010 * 0.01) = 16,500.10$$
 
-$$ticksPassed = ticksPassed + geometricExponentIncrementDistanceInTicks = 36000000 + 9000000 = 45000000$$
+$$ticksPassed = ticksPassed + geometricExponentIncrementDistanceInTicks =
+36000000 + 9000000 = 45000000$$
 
-$$totalPrice = totalPrice + maxPriceForCurrentAdditiveIncrementInTicks = 10000 + 90000 = 100000$$
+$$totalPrice = totalPrice + maxPriceForCurrentAdditiveIncrementInTicks =
+10000 + 90000 = 100000$$
 
-100000 is greater than 16,500.10. This means we must now find out how many additive tick in the currentAdditiveIncrementInTicks of -2 we must pass in order to reach 16,500.10.
+100000 is greater than 16,500.10. This means we must now find out how many
+additive tick in the currentAdditiveIncrementInTicks of -2 we must pass in
+order to reach 16,500.10.
 
-$$ticksToBeFulfilledByExponentAtCurrentTick = (desiredPrice - totalPrice) / currentAdditiveIncrementInTicks = (16500.10 - 100000) / 0.01 = -8349990$$
+$$ticksToBeFulfilledByExponentAtCurrentTick = (desiredPrice - totalPrice) /
+currentAdditiveIncrementInTicks = (16500.10 - 100000) / 0.01 = -8349990$$
 
-$$tickIndex = ticksPassed + ticksToBeFulfilledByExponentAtCurrentTick = 45000000 + -8349990 = 36650010$$
+$$tickIndex = ticksPassed + ticksToBeFulfilledByExponentAtCurrentTick =
+45000000 + -8349990 = 36650010$$
 
 Bob set his limit order at tick 36650010
 
-#### Consequences
+### Consequences
 
-This decision allows us to define ticks at spot prices that users actually desire to trade on, rather than arbitrarily defining ticks at .01% distance between each other. This will also make integration with UX seamless, instead of either
+This decision allows us to define ticks at spot prices that users actually
+desire to trade on, rather than arbitrarily defining ticks at .01% distance
+between each other. This will also make integration with UX seamless,
+instead of either:
 
 a) Preventing trade at a desirable spot price or
-b) Having the front end round the tick's actual price to the nearest human readable/desirable spot price
+b) Having the front end round the tick's actual price to the nearest
+  human readable/desirable spot price
 
-One draw back of this implementation is the requirement to create many ticks that will likely never be used. For example, in order to create ticks at 10 cent increments for spot prices greater than _$10000_, a $exponentAtPriceOne$ value of -5 must be set, requiring us to traverse ticks 1-3600000 before reaching _$10,000_. This should simply be an inconvenience and should not present any valid DOS vector for the chain.
+One draw back of this implementation is the requirement to create many ticks
+that will likely never be used. For example, in order to create ticks at 10
+cent increments for spot prices greater than _$10000_, a $exponentAtPriceOne$
+value of -5 must be set, requiring us to traverse ticks 1-3600000 before
+reaching _$10,000_. This should simply be an inconvenience and should not
+present any valid DOS vector for the chain.
```

Comment:
I don't think this makes much sense anymore after having a stable exponent at price one of -6